cmake/3rdParty/Platform/Mac/BuiltInPackages_mac.cmake

```diff
@@ -44,3 +43,4 @@ ly_associate_package(PACKAGE_NAME googlebenchmark-1.5.0-rev2-mac
 ly_associate_package(PACKAGE_NAME OpenSSL-1.1.1b-rev1-mac                           TARGETS OpenSSL                     PACKAGE_HASH 28adc1c0616ac0482b2a9d7b4a3a3635a1020e87b163f8aba687c501cf35f96c)
 ly_associate_package(PACKAGE_NAME qt-5.15.2-rev5-mac                                TARGETS Qt                          PACKAGE_HASH 9d25918351898b308ded3e9e571fff6f26311b2071aeafd00dd5b249fdf53f7e)
 ly_associate_package(PACKAGE_NAME libsamplerate-0.2.1-rev2-mac                      TARGETS libsamplerate               PACKAGE_HASH b912af40c0ac197af9c43d85004395ba92a6a859a24b7eacd920fed5854a97fe)
+ly_associate_package(PACKAGE_NAME zlib-1.2.11-rev1-mac                              TARGETS zlib                        PACKAGE_HASH 7fd8a77b3598423d9d6be5f8c60d52aecf346ab4224f563a5282db283aa0da02)
```

Comment:
nitpick: missing newline at the end of the file

Reply:
I'll update it.